Source/Core/Core/Core.cpp

```diff
@@ -176,7 +176,7 @@ void DisplayMessage(const std::string& message, int time_in_ms)
     return;
 
   // Actually displaying non-ASCII could cause things to go pear-shaped
-  for (const char& c : message)
+  for (const unsigned char& c : message)
   {
     if (!std::isprint(c))
```

Comment:
It would likely be preferable to include `<locale>` and change this to `std::isprint(c, std::locale::classic())` which doesn't have undefined behavior in this manner at all and is required to give a conclusive true or false.